elasticsearch/src/main/java/org/hibernate/search/elasticsearch/client/impl/JestClient.java

```diff
@@ -119,9 +119,23 @@ public void start(Properties properties, BuildContext context) {
 						),
 						TimeUnit.SECONDS
 				)
-				.gson( gsonService.getGson() )
-				.build()
+				.gson( gsonService.getGson() );
+
+		String username = ConfigurationParseHelper.getString(
+				properties,
+				CLIENT_PROP_PREFIX + ElasticsearchEnvironment.SERVER_USERNAME,
+				null
 		);
+		if ( !StringHelper.isEmpty( username ) ) {
```

Comment:
Ok, sorry. There is no particular check in Jest or the Apache HTTP Client, so I guess an empty string is propagated to ES as is. From my tests, ES rejects empty usernames:

```
$ curl -XGET ':password@localhost:9200/_cat?pretty'
{
  "error" : {
    "root_cause" : [ {
      "type" : "security_exception",
      "reason" : "missing authentication token for REST request [/_cat?pretty]",
      "header" : {
        "WWW-Authenticate" : "Basic realm=\"shield\" charset=\"UTF-8\""
      }
    } ],
    "type" : "security_exception",
    "reason" : "missing authentication token for REST request [/_cat?pretty]",
    "header" : {
      "WWW-Authenticate" : "Basic realm=\"shield\" charset=\"UTF-8\""
    }
  },
  "status" : 401
}
```